src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -54,7 +54,7 @@ Whatever method you choose, you need to sign up for a [GitHub account](https://g
 GitHub is at the heart of our whole process. When you propose a change to the docs site, GitHub manages the two different versions of a doc: the original version as it exists on **docs.newrelic.com** and the edited version that exists on your screen. With any edit you make, the Docs team reviews and approves it before it's published.  
 
 <Callout variant="tip">
-If you're a New Relic employee, we recommend joining the New Relic org on GitHub. To join the org, ping the @hero in our open source help channel.
+If you're a New Relic employee, we recommend joining the New Relic org on GitHub. To join the org, ping the @hero in our security tooling help channel.
```

Comment:
@boylea Can we put the actual channel here? Just to confirm, is it #help-security-tooling?